From 44ec88e68fdc49e5a86df38d5021fd608c645c42 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Wed, 11 Mar 2026 03:40:51 +0100
Subject: [PATCH] util: Add missing chafa_dep to hb-raster

---
 util/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/meson.build b/util/meson.build
index f5f9b7ccb61..a66a48b1d0d 100644
--- a/util/meson.build
+++ b/util/meson.build
@@ -93,7 +93,7 @@ if conf.get('HAVE_GLIB', 0) == 1
     hb_raster = executable('hb-raster', hb_raster_sources,
       cpp_args: cpp_args,
       include_directories: [incconfig, incsrc],
-      dependencies: [util_deps],
+      dependencies: [util_deps, chafa_dep],
       link_with: [libharfbuzz, libharfbuzz_raster],
       install: true,
     )
